### DISCUSSION TITLE
Blurring and overlay of image using Imagesharp library

### DISCUSSION THREAD

**Discussion by vindhya124** [SixLabors/ImageSharp#2246] [Q&A] [answered]
I need to share an image to facebook from my website. Facebook accepts images which is of 1.91:1 ratio so i need a create an image similar to below which avoids cropping when shared on facebook. Is there a way we can achieve blurred borders using imagesharp library?

**Comment by vindhya124** [comment, marked as answer]
I have achieved this using Imagesharp library and is there a way i can close the discussion?

**Reply by JimBobSquarePants** [reply]
Post the sample of how you did it and mark that as an answer. Then you can help others with the same question. 

In addition, someone here might be able to give you more tips.

**Reply by vindhya124** [reply]
Sure thing.

Below are the steps.
1. I resized the image to 1150 width and 600 height, blurred the background and overlayed my 800*600 image.

Below is code.
 image1.Mutate(i => i.Resize(800, 600, KnownResamplers.Lanczos3)); 

 image.Mutate(i => i.Resize(1150, 600, KnownResamplers.Lanczos3).BokehBlur().DrawImage(image1, new Point(180,0), opacity: 1f).Brightness(1.0f));

Below is the facebook debugger tool where we can test how our feed is shared.
https://developers.facebook.com/tools/debug/

And below is the image i am sharing to facebook.
![image](https://user-images.githubusercontent.com/38023844/194954688-6a11b96f-8082-4f09-b680-f751392aca34.png)
